### ISSUE TITLE
Support for this.data.actions to be a function that returns an array

### ISSUE THREAD

**Issue by dlmiles** [bomsy/grunt-regex-replace#9]
Support for this.data.actions to be a function that returns an array, as well as a raw array.

https://github.com/bomsy/grunt-regex-replace/blob/master/tasks/regex-replace.js#L19

Allowing:

```JavaScript
		// This will ensure the first 25 LINK element have closing /> form
		"regex-replace": {
			fixXhtmlLinkElements: {
				src: [ '**/*.html' ],
				actions: function() {
					var func = function(s) {
						if(arguments.length > 2) {
							var l = arguments[1].length;	// length of match0
							var ch = '';
							if(l > 0)
								ch = arguments[1].charAt(l - 1);	// get last char
							var insert = (ch === '/') ? '' : '/';	// decide if we need to insert '/' or not
							return arguments[1] + insert + arguments[2];
						} else {
							return arguments[0];
						}
					};

					var ary = [];
					for(var i = 1; i <= 25; i++) {
						var o = {
							name: 'link' + i,
							search: new RegExp('(<link\\s+[^>]*)(>)'),
							replace: func,
							flags: 'gi'
						};
						ary.push(o);
					}
					return ary;
				}
			}
		},

```

**Comment by bomsy** [comment]
Sorry have not being on this for a while. This is cool idea. Will add this.

**Comment by bomsy** [comment]
Published version 0.2.7 with this fix.
